comfy_cli/command/run.py

```
@@ -283,87 +780,192 @@ def log_node(self, type, node_id):
+        return {
+            "category": category,
+            "node_id": node_id,
+            "class_type": self.emitter.get_class_type(node_id),
+            "title": self.emitter.get_title(node_id),
+            "filename": filename,
+            "subfolder": subfolder,
+            "type": file_type,
+            "url": url,
+        }
 
     def on_message(self, message):
```

Pylint (4.0.5)

[convention] Missing function or method docstring

(C0116)